### ISSUE TITLE
DrRacket can't install the package

### ISSUE THREAD

**Issue by drasken** [sicp-lang/sicp#44]
Hi, 

I'm trying to folow SICP with DrRacket but when trying to install the package this error is returned

```

ssl-make-client-context: requested protocol not supported;
 SSL not available; check `ssl-load-fail-reason'
  requested: 'auto
  context...:
   /usr/share/racket/collects/openssl/mzssl.rkt:672:0: encrypt->method
   /usr/share/racket/collects/openssl/mzssl.rkt:747:0: make-raw-context
   /usr/share/racket/collects/openssl/mzssl.rkt:736:0: make-context
   /usr/share/racket/collects/openssl/mzssl.rkt:764:0: ssl-make-client-context
   /usr/share/racket/collects/openssl/mzssl.rkt:1061:0: ssl-secure-client-context
   /usr/share/racket/collects/openssl/mzssl.rkt:779:0: get-context
   /usr/share/racket/collects/openssl/mzssl.rkt:615:18
   /usr/share/racket/collects/ffi/unsafe/atomic.rkt:73:13
   /usr/share/racket/collects/openssl/mzssl.rkt:1590:0: wrap-ports
   /usr/share/racket/collects/racket/contract/private/arrow-val-first.rkt:555:3
   /usr/share/racket/collects/net/http-client.rkt:67:0: http-conn-open!
   /usr/share/racket/collects/net/http-client.rkt:274:0: http-conn-open
   /usr/share/racket/collects/racket/contract/private/arrow-val-first.rkt:555:3
   /usr/share/racket/collects/net/url.rkt:202:0: http://getpost-impure-port
   /usr/share/racket/collects/net/url.rkt:305:0: get-pure-port/headers
   /usr/share/racket/collects/racket/contract/private/arrow-val-first.rkt:555:3
   ...
```
don't know if is my problem or if I can bypass it by downloading directly here from github or whir raco command in the shell

**Comment by sorawee** [comment]
How do you install Racket? What OS are you using? Can you install other Racket packages (say, `raco pkg install threading`)?

**Comment by drasken** [comment]
Sorry, probably missed too much of the context.

 And apparently the problem is on my machine. Tryed the `raco pkg install threading` but still the same error in output.

Anyway, I installed racket on Ubuntu 22.04 with the `sudo apt -y install racket` command, now I just need to figure out what the problem is.

**Comment by sorawee** [comment]
According to https://github.com/racket/racket/issues/4306#issuecomment-1159614921, `apt-get install libssl-dev` should fix the problem. 

Subsequent comments indicate that this issue can also be fixed by upgrading to Racket 8.6 (the current latest version is 8.8, and we are releasing 8.9 soon). Assuming that you are currently using an old version of Racket, upgrading would be another possibility. The Racket PPA (https://launchpad.net/~plt/+archive/ubuntu/racket) seems to be defunct, but it does provide Racket 8.6, so you can also use that.

**Comment by drasken** [comment]
Thank you very much, I solved it. 

As you pointed out (I guiltily didn't notice), the version supported installed on Ubuntu with apt is really outdated, after looking on how to update I've that the snap instead is updated to the current version 8.8, so I just Installed with `sudo snap install racket  ` , than installed sicp and now it works just fine.

Didn't try `apt-get install libssl-dev` , I took the chance to use a newer version.
